mantidproject/mantid
# Add background scale to WANDPowderReduction

**To test:**
Look at [usage examples](http://docs.mantidproject.org/nightly/algorithms/WANDPowderReduction-v1.html#usage) and in lieu of background try subtracting silicon from itself with different background scales.
_e.g._
```python
silicon =LoadWAND('/HFIR/HB2C/IPTS-7776/nexus/HB2C_26506.nxs.h5')
vanadium=LoadWAND('/HFIR/HB2C/IPTS-7776/nexus/HB2C_26509.nxs.h5')

WANDPowderReduction(InputWorkspace=silicon,
                    CalibrationWorkspace=vanadium,
                    Target='Theta',
                    NumberBins=1000,
                    OutputWorkspace='silicon_powder')

WANDPowderReduction(InputWorkspace=silicon,
                    BackgroundWorkspace=silicon,
                    BackgroundScale=0.5,
                    CalibrationWorkspace=vanadium,
                    Target='Theta',
                    NumberBins=1000,
                    OutputWorkspace='silicon_powder_bkg')
```
You see that half the intensity has been subtracted as background.
![silicon_background_scaled](https://user-images.githubusercontent.com/5595210/39641387-f7ed02ac-4f9b-11e8-97cc-baf32863bfdf.png)


**Release Notes** 
*Does not need to be in the release notes and the algorithm was only added in this release*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
